lib/fileutils.rb

```diff
@@ -898,13 +953,13 @@ def copy_stream(src, dest)
   #   |   `-- src.txt
   #   `-- src1.txt
   #
-  # - <tt>force: true</tt> - attempts to force the move;
-  #   if the move includes removing +src+
-  #   (that is, if +src+ and +dest+ are on different devices),
-  #   ignores raised exceptions of StandardError and its descendants.
+  # Keyword arguments:
+  #
+  # - <tt>force: true</tt> - ignores raised exceptions of StandardError
```

Comment:
My understanding of the previous documentation is that in this case, the `:force` flag only affects cases where copying and then removing the original source is needed, and in this case it would apply to the removal phase only. If this is correct, I think the previous documentation is more accurate, because it doesn't imply that raised exceptions during the copy phase are swallowed.

Reply:
What should this say, then?

Reply:
I would say what it says now, because I'm not sure whether your change improves things or not. Except thet I'd probably remove the first sentence "attempts to force the move", because in my mind it's quite unclear what that means. So I'd go with:

```
  # - <tt>force: true</tt> - if the move includes removing +src+ (that is, if +src+ and +dest+ are on different devices), ignores raised exceptions of StandardError and its descendants.
```

Reply:
I've made it so.